mvc/src/main/java/web/org/springframework/web/bind/annotation/CustomRequestMappings.java

```diff
@@ -0,0 +1,26 @@
+package web.org.springframework.web.bind.annotation;
+
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+
+public enum CustomRequestMappings {
+
+    GET(GetMapping.class),
+    POST(PostMapping.class),
+    PUT(PutMapping.class),
+    DELETE(DeleteMapping.class),
+    PATCH(PatchMapping.class);
+
+    private final Class<?> annotation;
+
+    CustomRequestMappings(Class<?> annotation) {
+        this.annotation = annotation;
+    }
+
+    public static boolean isAnyMatch(Annotation annotation) {
+        Class<? extends Annotation> annotationType = annotation.annotationType();
+
+        return Arrays.stream(values())
+                .anyMatch(each -> each.annotation.equals(annotationType));
+    }
```

Comment:
```suggestion
    public static boolean isAnyMatch(Annotation annotation) {
        return Arrays.stream(values())
                .anyMatch(each -> each.annotation.equals(annotation.annotationType()));
    }
```
간단히 표현해도 괜찮지 않을까요? 물론 취향차이 입니다~